product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx

```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
 
 ## Installation on Kubernetes
 
+### Obtaining an EDB subscription token
+
+Installing EDB Postgres for Kubernetes requires an EDB Repos 2.0 token to gain access to the EDB private software repositories. 
+You can obtain the token by visiting the [EDB Repos website](https://www.enterprisedb.com/repos) and signing into your account.
+Your account profile page displays the token to use - by default it is obscured, click the "Show" button (an eye icon) to reveal it.
+
+Your token entitles you to access one of two repositories: standard or enterprise. 
+
+* Standard - Includes the operator and operands for EDB Postgres Extended.
+* Enterprise - Includes the operator and operands for EDB Postgres Advanced and EDB Postgres Extended.
+
+Export the appropriate value as an environmental variable EDB_SUBSCRIPTION_PLAN.
+
+```shell
+export EDB_SUBSCRIPTION_PLAN=enterprise
+```
+
+then export the Repos 2.0 token to an environment variable EDB_SUBSCRIPTION_TOKEN.
+
+```shell
+export EDB_SUBSCRIPTION_TOKEN=<your-token>
```

Comment:
You don't need to export the variables to use them in the same shell.

Suggested change
```suggestion
EDB_SUBSCRIPTION_TOKEN=<your-token>
```